README.md

```diff
@@ -162,6 +162,13 @@ server-tools: # Here we repeat server-tools, but no problem because it's a
     - html_text
 ```
 
+You can add all modules in a repo by using a `*`:
+
+```yaml
+website:
+    - "*"
```

Comment:
Hahaha ok so glob support was my first thought, which ended up being pretty damned difficult. In hindsight, I probably should have just made it a regex & replaced the `*` with `.*?` instead. Will give that a shot